modules/vaos/app/services/vaos/v1/fhir_service.rb

```diff
@@ -45,11 +45,21 @@ def search(query_string)
         query_string.blank? ? perform(:get, @resource_type.to_s) : perform(:get, "#{@resource_type}?#{query_string}")
       end
 
+      # The create interaction creates a resource based on the FHIR resource passed in the request body.
+      # The interaction is preformed by and HTTP POST command.
+      # http://hl7.org/fhir/dstu2/http.html#create
+      #
+      # @request ActionDispatch Request containing POST request values.
+      #
+      def create(request)
```

Comment:
Could `create()` take a hash rather than the entire request? (move the body.read up to the controller). Most likely that it's always a request, but in the future the service might be called by something other than a controller.